src/main/resources/application.yaml

```diff
@@ -291,6 +305,7 @@ robotics.email.message: ${ROBOTICS_EMAIL_MESSAGE:Please find attached the roboti
 robotics.schema.resource.location: /schema/sscs-robotics.json
 
 feature.bundled_letters_on: ${BUNDLED_LETTERS_ON:false}
```

Comment:
very minor thing, you can do:
`feature:
    bundled_letters_on:
    lettes_on:
`